### PR TITLE
PC-7896: add github actions

### PR DESCRIPTION
**Pull request by vfrmn**

### CODE REVIEW

**Review by dzubchik — CHANGES_REQUESTED** [review]
update links to openfintech repository from own

File: etc/deploy.sh [hunk 1]
```diff
@@ -3,27 +3,26 @@
...
+  git config --global user.name "Github Actions"
 }
 
 push_files() {
-  git push origin master
+  git add .
+  if ! (git status | grep -q "nothing to commit"); then
+    git commit --message "Github actions build: ${BUILD_NUMBER}"
+    git push origin master
+    git push origin master
```
Comment (loperd) [inline etc/deploy.sh:15]: rm

Comment (vfrmn) [reply etc/deploy.sh]: +

Comment (vfrmn) [reply etc/deploy.sh]: \+

**Review by loperd — APPROVED** [review]

**Pull request merged**